client/my-sites/media-library/content.jsx

```diff
@@ -182,7 +226,11 @@ const MediaLibraryContent = React.createClass( {
 
 	renderMediaList: function() {
 		if ( ! this.props.site ) {
-			return <MediaLibraryList key="list-loading" />;
+			return <MediaLibraryList key="list-loading" filterRequiresUpgrade={ this.props.filterRequiresUpgrade } />;
```

Comment:
Yep, if you're sandboxing the API it can be a bit slow, and this leads to the situation where the modal is displayed before the site data has loaded, causing the missing `isRequired` PropType to fail - it got annoying so I quickly fixed it!